src/mon/MgrMap.h

```diff
@@ -209,6 +234,22 @@ class MgrMap
       }
       DECODE_FINISH(p);
     }
+    void dump(ceph::Formatter *f) const {
+      f->dump_unsigned("gid", gid);
+      f->dump_string("name", name);
+      f->open_array_section("available_modules");
+      for (auto& i : available_modules) {
```

Comment:
nit: `const auto&`.

Reply:
using encode_json will reduce the number of lines